pages/api/email-esrf.ts

```diff
@@ -1,19 +1,25 @@
 import { NextApiRequest, NextApiResponse } from 'next'
-import logger from '../../lib/logger'
 import { EmailEsrfApiRequestBody } from '../../lib/types'
+import { getLogger } from '../../logging/log-util'
+
+const logger = getLogger('email-esrf')
 
 export default async function handler(
   req: NextApiRequest,
   res: NextApiResponse<string>
 ) {
   if (req.method !== 'POST') {
-    res.status(405).send(`Invalid request method ${req.method}`)
-    return
+    logger.error(`Status 405: Invalid request method ${req.method}`)
```

Comment:
I would log this as either `warn` or `debug` because this isn't an application error.